### ISSUE TITLE
ng add @angular/material throws error

### ISSUE THREAD

**Issue by marlowp** [angular/material2#12153]
#### Bug, feature request, or proposal:
Bug

#### What is the expected behavior?
Running `ng add @angular/material` should work

#### What is the current behavior?
It throws the following error:
```
ng add @angular/material
Installing packages for tooling via npm.
npm WARN @angular/material@6.3.3 requires a peer of @angular/cdk@6.3.3 but none is installed. You must install peer dependencies yourself.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.4 (node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.4: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})

+ @angular/material@6.3.3
added 2 packages from 1 contributor and audited 21840 packages in 7.296s
found 13 vulnerabilities (9 low, 4 high)
  run `npm audit fix` to fix them, or `npm audit` for details
Installed packages for tooling via npm.
Cannot read property 'startTag' of undefined
```

#### What are the steps to reproduce?
```
npm install -g @angular/cli
ng new my-dream-app
cd my-dream-app
ng add @angular/material
```

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
```
Angular CLI: 6.0.8
Node: 10.6.0
OS: win32 x64
Angular: 6.0.7
... animations, common, compiler, compiler-cli, core, forms
... http, language-service, platform-browser
... platform-browser-dynamic, router

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.6.8
@angular-devkit/build-angular     0.6.8
@angular-devkit/build-optimizer   0.6.8
@angular-devkit/core              0.6.8
@angular-devkit/schematics        0.6.8
@angular/cli                      6.0.8
@angular/material                 6.3.3
@ngtools/webpack                  6.0.8
@schematics/angular               0.6.8
@schematics/update                0.6.8
rxjs                              6.2.1
typescript                        2.7.2
webpack                           4.8.3
```

#### Is there anything else we should know?
Manually installing `@angular/cdk` does not fix the issue.

**Comment by DevVersion** [comment]
Duplicate of #12027

**Comment by Razenbull** [comment]
@DevVersion it is a duplicate indeed but we're using ```@angular/material@6.3.3``` and the fix [#12028](https://github.com/angular/material2/pull/12028/commits/735c15b539671a4651aa75fc432b13b3499509a9) has already been merged in master branch but I still the error ```Cannot read property 'startTag' of undefined```

**Comment by DevVersion** [comment]
Yeah but the PR has been merged yesterday and is not released yet.
